components/atoms/TranslatedText.tsx

```diff
@@ -0,0 +1,12 @@
+import { Typography, TypographyProps  } from '@mui/material';
+import React from 'react';
+
+// TS error for components
+// https://github.com/mui-org/material-ui/issues/19512#issue-558587710
+
+type Props =  TypographyProps & { children: string, component?: any };
```

Comment:
Why `component`?
If it's a component, maybe it should be JSXElement or React.ReactNode.
If it's the `component` prop for the Typography, should it be a string?

Reply:
Original component type is `any`, mirrored here